koza/model/config/source_config.py

```diff
@@ -151,13 +151,11 @@ class SourceFileConfig:
     name: str
     files: List[Union[str, Path]]
     format: FormatType = FormatType.csv
-    path: Path = None
     standard_format: StandardFormat = None
     file_metadata: DatasetDescription = None
     columns: List[Union[str, Dict[str, FieldType]]] = None
     node_properties: List[str] = None
     edge_properties: List[str] = None
-    required_properties: List[str] = None
```

Comment:
do we need this for the json reader or am I misremembering?

Reply:
Yeah....I should have left that one in. It didn't look like we've used it, and my IDE thought it didn't have any usages, but it definitely needs to go back in.